### PR TITLE
ability to set server_ssl_crl setting with a CA-provided cert instead of puppets

### PR DESCRIPTION
**Pull request by smbambling**

### CODE REVIEW

**Comment by mmoll** [comment]
looks legit :+1:

File: manifests/init.pp [hunk 1]
```diff
@@ -117,6 +117,8 @@
 #
 # $server_ssl_key::           Defines Apache mod_ssl SSLCertificateKeyFile setting in Foreman vhost conf file.
 #
+# $server_ssl_crl::           Defines the Apache mod_ssl SSLCARevocationFile settign in Foreman vhost conf file.
```
Comment (ekohl) [inline manifests/init.pp]: s/settign/setting/

**Comment by ekohl** [comment]
Other than the minor typo :+1:. Note that we prefer squashed commits but don't mind squashing for you.

**Comment by smbambling** [comment]
Typo fix and commits squashed.

**Comment by domcleal** [comment]
Merged, thanks @smbambling!

**Pull request merged**